projects/core/src/product/occ/product-reviews.service.ts

```diff
@@ -46,7 +46,7 @@ export class ProductReviewsLoaderService extends ProductOccService {
       });
 
     if (maxCount && maxCount > 0) {
-      url += `?maxCount=${maxCount}`;
+      url += `${url.indexOf('?') === -1 ? '?' : '?'}maxCount=${maxCount}`;
```

Comment:
eh... 

`${url.indexOf('?') === -1 ? '?' : '&'}`

Reply:
for readability I'd make it 2 lines, first concatenate the operator and then the parameter